dev/F5WebApi.Net31/F5WebApi.Net31.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <ItemGroup>
+    
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights.Kubernetes" Version="3.0.0-private-*" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
Given that .NET 3.1. is out of support, I would not add this example. For the rest of the review, I assume the sample will be removed and I will not review other files that are related to the sample.

Reply:
I understand. This sample has a different purpose - it is a manual test tool to help cover `.NET Core 3.1` code branch - considering most of the development will be there for `6.x+`.
But anyway, per our discussion, it probably makes more sense to take the other approach. I'll submit another PR to address the issue.